config/initializers/application_insights.rb

```diff
@@ -4,5 +4,7 @@
   Rails.application.configure do
     buffer_size = 1
     config.middleware.use ApplicationInsights::Rack::TrackRequest, instrumentation_key, buffer_size
+
+    ApplicationInsights::UnhandledException.collect(app_insights_key)
```

Comment:
Does this automatically alert us, or do we need to set up some more alerts?

Reply:
No, this just populated Application Insights. We will need to set up any alerts in there.

Reply:
Cool 👍